File: Hard/Password Validation/Password Validation.cs
using System;

class Test
{

    static void Main()


    {

        string passWords = Console.ReadLine();

        bool b = ValidatePassword(passWords);
        if (b)
        {
            Console.WriteLine("Strong");
        }
        else
        {
            Console.WriteLine("Weak");
        }

        Console.ReadKey();

    }

    static bool ValidatePassword(string passWords)

    {

        int validSpecialCharacters = 0;
        //int validAlphabets = 0;
        int validNumbers = 0;

        //foreach (char c in passWord)

        //{

        //    if (c >= 'a' && c <= 'z')

        //    {

        //        validConditions++;

        //        break;

        //    }

        //}

        //foreach (char c in passWord)

        //{

        //    if (c >= 'A' && c <= 'Z')

        //    {

        //        validConditions++;

        //        break;

        //    }

        //}

        //if (validConditions == 0) return false;

        foreach (char c in passWords)
        {
            if (c >= '0' && c <= '9')
            {
                validNumbers++;
            }
        }
        if (validNumbers >= 2)
        {
            char[] special = { '!', '@', '#', '$', '%', '&', '*' }; // or whatever
            foreach (char c in passWords)
            {
                if (Array.Exists(special, element => element == c))
                {
                    validSpecialCharacters++;
                }
            }
        }
        return (passWords.Length >= 7 && validSpecialCharacters >= 2 && validNumbers >= 2);
    }

}

File: Hard/Password Validation/Password Validation.cpp
#include <iostream>
using namespace std;

int main() {
int a=0,b=0,c=0,d=0;
string pswrd;
getline(cin,pswrd);
a=pswrd.length();
for(; b<a; b++){
    if(pswrd[b]=='$'||pswrd[b]=='&'||pswrd[b]=='@'||pswrd[b]=='%'||pswrd[b]=='#'||pswrd[b]=='*'||pswrd[b]=='!'){
    c++;}
        if(pswrd[b]=='1'||pswrd[b]=='2'||pswrd[b]=='3'||pswrd[b]=='4'||pswrd[b]=='5'||pswrd[b]=='6'||pswrd[b]=='7'||pswrd[b]=='8'||pswrd[b]=='9'||pswrd[b]=='0'){
    d++;}
}
    if(c>=2 && d>=2 && a>=7){
        cout << "Strong";
    
}else {
    cout << "Weak";
}
    return 0;
}
